apps/meteor/client/hooks/useLicense.ts

```diff
@@ -0,0 +1,32 @@
+import { CloudRegistrationStatus, ILicense } from '@rocket.chat/core-typings';
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+
+export const useLicense = (): {
```

Comment:
As we talked on a DM, move this to 2 separate hooks.

Reply:
Done!